pages/common/mosh.md

```diff
@@ -0,0 +1,33 @@
+# mosh
+
+> Mobile Shell (`mosh`) is remote terminal application that allows roaming.
+> `mosh` is a more robust and responsive replacement for SSH.
+> Visit <https://mosh.org> for more information.
```

Comment:
We always restrict the heading to 2 lines. Either delete this line or the one above this. Or merge them to a single line.